pkg/dyninst/gosym/symtab.go

```diff
@@ -0,0 +1,1187 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
+
+// Package gosym provides SymbolTable and PcLnTable utilities.
```

Comment:
nit: note that it's inspired by the other gosym library but say why it's better. Probably also include a link